io.github.SamuelSchlemperSchlemuel.SingWriter.json

```diff
@@ -0,0 +1,39 @@
+{
+    "id" : "io.github.SamuelSchlemperSchlemuel.SingWriter",
```

Comment:
app-id doesn't match github's

Reply:
this is the blocker.

https://docs.flathub.org/docs/for-app-authors/requirements#application-id